ui/packages/consul-ui/app/components/consul/datacenter/selector/index.hbs

```diff
@@ -17,6 +17,9 @@
         @onchange={{action (mut @dcs) value="data"}}
       />
         <panel.Menu as |menu|>
+          <p class="dcs-message">
+            Datacenters shown in this dropdown are available through WAN Federation.
+          </p>
```

Comment:
I think this adds a `p` tag into the `ul`. If you move it one level up out of the menu though, its parent will be a `div` instead, which is allowed.